src/MailingListBundle/Entity/MailingList.php

```diff
@@ -54,6 +54,24 @@ class MailingList
     protected $subscriptions;
 
     /**
+     * @var string
+     *
+     * @ORM\Column(name="provider", type="string", length=255, nullable=true)
```

Comment:
I don't think the provider should be nullable. It think it's better to require setting the provider.